latency-tests-analysis/scripts/generate_latency_report.py

```diff
@@ -64,7 +64,14 @@ def generate_adoc(output):
         vm_names = [file.split("ts_")[1].split(".txt")[0]
                     for file in vm_result_files]
 
-        adoc_file.write("== Latency tests\n")
+        adoc_file.write(
+                """== Latency tests
+                   === Test setup
+                   A series of SV packets is sent from the publisher over the local network using a switch. These packets are received by a VM on the SEAPATH hypervisor. SVs are received on a PCI passthrough interface, which means that the physical network card is directly assigned to the VM, enabling direct access to packets without passing through the virtualization layer, thus ensuring low latency.
```

Comment:
- Remove mention of the switch (I don't think it is relevant).
- Don't explain PCI-passthrough (We will put that in the wiki)
- add how many paquets were sent ( we could calculate that automatically)